pages.es/common/pamtogif.md

```diff
@@ -0,0 +1,17 @@
+# pamtogif
+
+> Convierte una imagen Netpbm en una imagen GIF no animada.
+> Vea también: `giftopnm`, `gifsicle`.
+> Más información: <https://netpbm.sourceforge.net/doc/pamtogif.html>.
+
+- Convierte una imagen Netpbm en una imagen GIF no animada:
+
+`pamtogif {{ruta/a/imagen.pam}} > {{ruta/a/imagen_de_salida.gif}}`
+
+- Marca el color especificado como transparente en el archivo GIF de salida:
+
+`pamtogif -transparent {{color}} {{ruta/a/imagen.pam}} > {{ruta/a/imagen_de_salida.gif}}`
+
+- Incluye el texto especificado como comentario en el archivo GIF de salida:
+
+`pamtogif -comment "{{¡Hola Mundo!}}" {{ruta/a/imagen.pam}} > {{ruta/a/imagen_de_salida.gif}}`
```

Comment:
I was worried for a bit that this line could accidentally perform history expansion. However, at least in ZSH that won't happen unless characters are added to the placeholders.